### ISSUE TITLE
pip installation failed

### ISSUE THREAD

**Issue by bguil** [proycon/foliapy#16]
On a CentOS linux distribution, the installation command:

> pip3 install folia

fails with the following output:

```
Collecting folia
  Downloading https://files.pythonhosted.org/packages/28/a6/bdd6d032eefd491ac15ca1ae4682592baf0c0088ae522c9631aa27068266/FoLiA-2.2.3.tar.gz (172kB)
    100% |################################| 174kB 4.5MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-dpnvdhp9/folia/setup.py", line 34, in <module>
        long_description=read('README.rst'),
      File "/tmp/pip-build-dpnvdhp9/folia/setup.py", line 19, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
      File "/usr/lib64/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 565: ordinal not in range(128)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-dpnvdhp9/folia/
```

Any idea of what's going wrong?

**Comment by proycon** [comment]
This is probably caused by your system's locale not being set to a UTF-8 locale. I released a new version just now (2.2.4) that should hopefully be more robust and install even in that situation.

Can you try if it works now?

**Comment by bguil** [comment]
It works.

thanks for your prompt help!
